### PR TITLE
Spectrum tokens 17 card

### LINKED ISSUE #811: Check card against DL and VRT
Check [card](https://github.com/adobe/spectrum-css/tree/spectrum-tokens-17/components/card)
- [ ] matches DL
- [ ] check VRT passes
- [ ] check and update migration guides [example](https://github.com/adobe/spectrum-css/blob/f58d8e071e5cce8db653e61c5a09b7690c236f63/components/pagination/metadata/pagination-explicit.yml#L3)

### PR DESCRIPTION
**Pull request by jianliao**
## Description
Fixed #811 
1. Text line-height change caused by typography - expected
2. `More` icon size change - fixed
3. Card title font-color and font-weight change to match the DL

## How and where has this been tested?
 - **How this was tested:** 
 - **Browser(s) and OS(s) this was tested with:** 

## Screenshots

Line-height change caused by the text in footer and card title font color/weight change.
image

DL:
image

## To-do list

- [ ] If my change impacts other components, I have tested to make sure they don't break.
- [ ] If my change impacts documentation, I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).

- [x] This pull request is ready to merge.

### CODE REVIEW

**Comment by GarthDB** [comment]
I chatted with @NateBaldwinDesign why to find out why we don't have more typography tokens for `spectrum-Card-title` and `spectrum-Card-subtitle` and it is so the products can have more control over which size title and subtitle to use with the `spectrum-Heading` and `spectrum-Detail` components.

File: components/card/package.json [hunk 1]
```diff
@@ -30,6 +31,7 @@
     "@spectrum-css/component-builder": "^1.2.0",
     "@spectrum-css/icon": "^2.1.1",
     "@spectrum-css/quickaction": "^3.0.0-beta.3",
+    "@spectrum-css/typography": "^2.1.3",
```
Comment (lazd) [inline components/card/package.json:34]: Since this is in deps, we don't need it in devDeps too
```suggestion
```

File: components/card/skin.css [hunk 2]
```diff
@@ -31,7 +31,7 @@ governing permissions and limitations under the License.
   }
 
   .spectrum-Card-title {
-    color: var(--spectrum-alias-text-color);
+    color: var(--spectrum-global-color-gray-900);
```
Comment (lazd) [inline components/card/skin.css:34]: Any reason for this? I'd rather see an alias than a global, and it seems that maybe this was changed? I see

```
  --spectrum-alias-text-color: var(--spectrum-global-color-gray-800);
  --spectrum-alias-text-color-hover: var(--spectrum-global-color-gray-900);
```

Comment (lazd) [reply components/card/skin.css:34]: Perhaps this should be `gray-800`?

Comment (GarthDB) [reply components/card/skin.css:34]: Sorry, I should have removed that as well, one sec, and I'll update title and subtitle to just use the Heading and Detail components and delete those.

Comment (jianliao) [reply components/card/skin.css]: @GarthDB I just removed it.

Comment (GarthDB) [reply components/card/skin.css]: @jianliao beat me to it.

File: components/card/package.json (hunk 1 above)
Comment (GarthDB) [reply components/card/package.json:34]: That's `peerDependencies`, not `dependencies`.

**Pull request merged**